content/docs/0.8.0/users-guide/default-disk-and-node-config.md

```diff
@@ -0,0 +1,42 @@
+# Default disks and node configuration
+
+## Overview
+
+This feature allows the user to customize the default disks and node configurations in Longhorn for newly added nodes **using Kubernetes label and annotation**, instead of using Longhorn API or UI.
+For more details, see [this enhancement doc](https://github.com/longhorn/longhorn/blob/v0.8.1/enhancements/20200319-default-disks-and-node-configuration.md).
+
+## Usage
+
+### Add node tags for new nodes
+
+1. Scale up the Kubernetes cluster. Then the newly added nodes contain no node tag.
+2. Add the annotations to the new Kubernetes nodes. 
+    - The annotation format is `node.longhorn.io/default-node-tags: <node tag list with JSON string format>`. For example:
+    ```
+    node.longhorn.io/default-node-tags: '["fast","storage"]'
+    ``` 
+3. Wait for Longhorn syncing the node tag automatically.
+
+### Customize default disks for new nodes
+
+1. Enable the Longhorn setting `Create Default Disk on Labeled Nodes` to prevent the default disk to be created automatically on new nodes. 
```

Comment:
`to enable the feature. After the feature is enabled, the default disk will not be automatically created on the new nodes unless a certain label was set.`